src/hotspot/share/opto/loopPredicate.cpp

```diff
@@ -94,10 +94,11 @@ void PhaseIdealLoop::register_control(Node* n, IdealLoopTree *loop, Node* pred,
 //                                           uncommon_trap
 //
 //
-// We will create a region to guard the uct call if there is no one there.
-// The continuation projection (if_cont) of the new_iff is returned which
-// is an IfTrue projection. This code is also used to clone predicates to cloned loops.
-IfTrueNode* PhaseIdealLoop::create_new_if_for_predicate(ParsePredicateSuccessProj* parse_predicate_success_proj,
+// We will create a region to guard the uct call if there is no one there. The continuation projection (if_cont) of the
+// new_iff is returned which is an IfTrue projection. This code is also used to clone predicates to cloned loops.
+// 'rewire_uncommon_proj_phi_inputs' should be set to the non-default value 'true' when called for a false-path loop
+// during Loop Unswitching.
```

Comment:
Just nitpick. Can you return back length of comment's lines?

Reply:
Yes sure, done in new commit.